pkg/codegen/testing/test/testdata/transpiled_examples/cue-random-pp/dotnet/dotnet.csproj

```diff
@@ -1,14 +1,15 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>net7.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
-  </PropertyGroup>
-
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Why are we changing the target framework here?

Reply:
Probably a rebase side effect. This was bumped in #15373 and was rolled back when I ran `PULUMI_ACCEPT=true make test_codegen_dotnet`. I'm also not sure why it added `<PackageReference Include="Pulumi" Version="3.59.0" />`.